includes/admin/class-wc-stripe-settings-controller.php

```
@@ -232,14 +232,11 @@ public function admin_scripts( $hook_suffix ) {
 			&& ! $is_ap_enabled
 			&& 'yes' === get_option( 'wc_stripe_show_ocs_only_banner', 'no' );
 
-		$is_checkout_sessions_available      = false;
-		$adaptive_pricing_unavailable_reason = 'disabled';
-		if ( WC_Stripe_Helper::is_checkout_sessions_available() ) {
-			$adaptive_pricing_unavailable_reason = WC_Stripe_Helper::get_adaptive_pricing_account_unavailable_reason();
-			if ( null === $adaptive_pricing_unavailable_reason ) {
-				$is_checkout_sessions_available = true;
-			}
+		$adaptive_pricing_unavailable_reason = WC_Stripe_Helper::get_adaptive_pricing_account_unavailable_reason();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'admin_scripts'. (undefined)

(StaticAccess)